apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -254,21 +260,49 @@ func (so *ScaledObject) GetHPAMaxReplicas() int32 {
 	return defaultHPAMaxReplicas
 }
 
+// GetDefaultHPAMaxReplicas returns defaultHPAMaxReplicas
+func (so *ScaledObject) GetDefaultHPAMaxReplicas() int32 {
+	return defaultHPAMaxReplicas
+}
+
+// CheckReplicasNotNegative checks that Min/Max ReplicaCount defined in ScaledObject are not negative
+func (so *ScaledObject) CheckReplicasNotNegative(replicas ...int32) bool {
+	for _, replica := range replicas {
+		if replica < 0 {
+			return false
+		}
+	}
+	return true
```

Comment:
and here too, the pointer receiver is ignored. But I think the code can be made even cleaner without this function entirely

Reply:
> and here too, the pointer receiver is ignored. But I think the code can be made even cleaner without this function entirely

ok